opencti-platform/opencti-graphql/src/schema/attribute-definition.ts

```diff
@@ -53,7 +57,7 @@ export type EnumAttribute = { type: 'string', format: 'enum', values: string[] }
 export type VocabAttribute = { type: 'string', format: 'vocabulary', vocabularyCategory: string } & BasicDefinition;
 export type JsonAttribute = { type: 'string', format: 'json', multiple: false, schemaDef?: Record<string, any> } & BasicDefinition;
 export type FlatObjectAttribute = { type: 'object', format: 'flat' } & BasicDefinition;
-export type ObjectAttribute = { type: 'object', format: 'standard' } & BasicObjectDefinition;
+export type ObjectAttribute = { type: 'object', format: 'standard', sortBy?: ObjectSorting } & BasicObjectDefinition;
```

Comment:
`NestedObjectAttribute` does not need this sortBy property also?

Reply:
Yes good point